docs/streams/quickstart.html

```diff
@@ -40,10 +40,10 @@
 final Serde&lt;String&gt; stringSerde = Serdes.String();
 final Serde&lt;Long&gt; longSerde = Serdes.Long();
 
-// Construct a `KStream` from the input topic "streams-wordcount-input", where message values
+// Construct a `KStream` from the input topic "streams-plaintext-input", where message values
```

Comment:
Didn't have anywhere to put this, but should we also update line 35 to refer to the Quickstart code? it currently points to examples